src/metabase/search/config.clj

```diff
@@ -143,6 +143,21 @@
     :native-query   {:type :native-query, :context-key :search-native-query}
     :verified       {:type :single-value, :supported-value? #{true}, :required-feature :content-verification}}))
 
+(def ^:private filter-defaults-by-context
+  {:default         {:archived               false
```

Comment:
I dislike how `:default` is used in two different senses here, but I don't have any ideas to avoid it.